services/network_service_test.go

```diff
@@ -30,7 +30,7 @@ var (
 	defaultNetworkOptions = &types.NetworkOptionsResponse{
 		Version: &types.Version{
 			RosettaVersion:    types.RosettaAPIVersion,
-			NodeVersion:       "3.1.0",
+			NodeVersion:       "3.2.0",
```

Comment:
maybe we could start using .env variables here to reduce the places where we need to update version, `os.Getenv()` I can do a couple of test tomorrow regarding this